<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeDialog</class>
 <widget class="QWidget" name="ModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Mode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="KIconButton" name="ibIcon">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>76</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="KLineEdit" name="leName"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="KSeparator" name="kseparator"/>
   </item>
   <item row="3" column="2">
    <widget class="KComboBox" name="cbButtons"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Button:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="cbSetDefault">
     <property name="text">
      <string>Set mode as default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
